tests/unittests/Foundation/NSDictionaryTests.mm

```diff
@@ -160,6 +160,172 @@
     EXPECT_OBJCEQ(@"world", dict[@"hello"]);
 }
 
+TEST(NSDictionary, ReadWriteURL) {
+    NSURL* url = [NSURL fileURLWithPath:@"dictionaryTestData.txt" isDirectory:NO];
```

Comment:
No, we have some issues with temp folders in unit tests, but I _should definitely_ be deleting this file 🐋